code/android-core-library/src/legacy/test-common/java/com/adobe/marketing/mobile/BaseTest.java

```diff
@@ -30,12 +30,10 @@
 
 public abstract class BaseTest {
 
-	MockEventHubUnitTest eventHub;
```

Comment:
Wondering if it's better to just delete this class?

Reply:
I am working on moving Variant related class and will do it as part of next PR.